tools/log-collector/helper.go

```diff
@@ -145,33 +145,28 @@ func filterCSV(csvObjects unstructured.UnstructuredList) unstructured.Unstructur
 	return filteredCSVObject
 }
 
-// filterCRD returns list of crds created by given set of groups
-func filterCRD(crdObjs unstructured.UnstructuredList) (unstructured.UnstructuredList, error) {
-	crdFilterGroup := []string{TriliovaultGroup, SnapshotStorageGroup, CsiStorageGroup}
+// filterRelatedCRD returns list of crds created by given set of groups
+func filterRelatedCRD(crdObjs unstructured.UnstructuredList) (unstructured.UnstructuredList, error) {
```

Comment:
again RelatedCRD? what would this mean to anyone reading this name
just add names of CRD grps in func name